src/run.ts

```diff
@@ -106,10 +113,14 @@ export async function runPublish({
 
     await Promise.all(
       releasedPackages.map((pkg) =>
-        createRelease(octokit, {
-          pkg,
-          tagName: `${pkg.packageJson.name}@${pkg.packageJson.version}`,
-        })
+        createRelease(
+          octokit,
+          {
+            pkg,
+            tagName: `${pkg.packageJson.name}@${pkg.packageJson.version}`,
+          },
+          skipCreateGithubRelease
```

Comment:
It imho makes more sense to call `createRelease` conditionally, rather than to skip releasing within this function.

Reply:
The whole content of the `createRelease` function is about creating the Github release, so I used a `guarded if`, but perhaps, putting the whole thing in an `if` block is a bit more future proof 😅